src/ripple/app/misc/TxQ.h

```diff
@@ -267,18 +402,36 @@ class TxQ
             };
         }
 
+        /** Use the number of transactions in the current open ledger
+            to compute the fee level a transaction must pay bypass the
+            queue.
+
+            @param view Current open ledger.
+            @param txCountPadding Optional number of "extra" transactions
+                to assume are in the ledger. Can be used to determine a
+                padded fee a transaction can pay to beat races with other
+                transactions.
```

Comment:
This comes back to the early days of the `TxQ` and an experimental, undocumented feature that really should be updated someday.

Long story short, if using `sign` or the sign-and-submit mode of `submit`, the fee will be populated with the exact value of the current open ledger fee. But there was concern (I don't remember if it came from a customer or my own imagination), that if the user _really_ wanted to get into the open ledger, but the transaction lost a race into the open ledger with another transaction paying the same fee, it would be queued. But rather than suggesting a fixed padding amount (which is probably what most people are doing), I added this option to calculate the open ledger fee as if there were X more transactions "padded" in the open ledger, thus increasing your chances of making it in. In retrospect it all sounds very silly. As far as I know, no other callers make use of padding, nor are any users using this option.

See also: RIPD-1177

Reply:
Is it fair to run the experiment of removing the `txCountPadding` argument on `getMetrics()` and `scaleFeeLevel()` to see if that argument is in use?  If no one is using it perhaps we can remove it?

Reply:
I already know the argument is in use from the user-facing undocumented feature. https://github.com/ximinez/rippled/blob/txq_doc/src/ripple/rpc/impl/TransactionSign.cpp#L705

I think the best course of action is to either remove that undocumented feature, or get it formalized and documented. I'l leaning towards the former, but I think it's out of scope for the PR either way.

Reply:
Yes, I agree it's out of scope for this pull request.  And also, yes, in some future PR I'd be very much in favor of removing the undocumented feature.  Things are complicated enough even without that undocumented feature.